### ISSUE TITLE
Frontend UI

### ISSUE THREAD

**Issue by avaer** [webaverse/xrpackage#38]
We should add a frontend UI for loading packages from XR. I imagine this being a palette similar to tilt brush, anyland, or supercraft. Except, instead of dealing in primitives or tools, its only job would be to drop packages into the world.

Not that there is a ton of precendent for doing this and we could probably piece it together from other Webaverse/Exokit open source repositories.

The closest thing to this is probably https://github.com/exokitxr/exokit-browser.

A place in the current codebase it might fit is "Edit mode" (`edit.html`).

**Comment by shu8** [comment]
Is this done with the 'Packages' panel now, or was this issue suggesting something else?

**Comment by avaer** [comment]
No, this means more a drag-and-drop 3D palette in VR. Some of this is in `master` already (https://github.com/webaverse/xrpackage-site/blob/fe399686533d22b599a18a062de9e7dc5288547b/edit.js#L12), but it needs a solid user experience and performance optimization pass before we flag it as enabled for everyone.
